sql/execution_pipeline_impl.cc

```diff
@@ -450,7 +525,7 @@ outputAdded() const {
 
 SubSelectExecutor::
 SubSelectExecutor(std::shared_ptr<BoundPipelineElement> boundSelect,
-                  const BoundParameters & getParam)
```

Comment:
My feeling is that we should make use of either a sparse matrix or tabular dataset for the sub selects; we're getting closer and closer towards reimplementing them anyway.